roles/telemetry_autoscaling/tasks/post_teardown.yml

```
@@ -0,0 +1,30 @@
+---    
+- name: Check that the stack exists
+  ansible.builtin.command:
+    cmd: |
+      oc rsh openstackclient openstack stack show {{ stack_name }}
+  ignore_errors: true
+  register: output
+
+- name: Remove the stack if it exists
+  ansible.builtin.command:
+    cmd: |
+      oc rsh openstackclient openstack stack delete {{ stack_name }}
+  when: output.rc == 0
+
+- name: Check whether a stack already exists with the same name
```

The name of this should be updated. It is mis-represented currently.

```suggestion
- name: Wait until the stack is deleted
```